### ISSUE TITLE
Cannot scale down

### ISSUE THREAD

**Issue by jacobtomlinson** [dask/daskernetes#27]
This might be related to #5 but when manually scaling a cluster I cannot remove workers by scaling to a lower number.

![image](https://user-images.githubusercontent.com/1610850/35444084-4968b9e4-02a5-11e8-94af-42f85ed1369f.png)

I've created a simple cluster. Used the widget to scale up to two. But when changing the number to zero or one and clicking scale again nothing happens. I'm having to manually clean up those pods with `kubectl`.

**Comment by jacobtomlinson** [comment]
I'm currently calling the following to clean up.

```
from daskernetes.core import cleanup_pods

cleanup_pods(cluster.namespace, cluster.worker_labels)
```

I see in `KubeCluster.__exit__` this is already being called but not in `KubeCluster.close()`. Perhaps it should be?

**Comment by mrocklin** [comment]
This has been resolved in #39
